fp.js

```diff
@@ -6,10 +6,16 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
+ /*
 people.forEach(function(person){
   peoplesAges.push(person.age)
 });
-
+*/
+  people.map(people =>{
```

Comment:
`.map` returns a new array containing a collection of things that match its `return` statement. In this case, you want to use map to generate an array containing only names, so the solution looks something like this...

```js
var people = {
  // data goes here
}

var peopleAges = people.map(person => {
  return person.age
})
```

This avoids the need to initialize an empty array and have to push items into it.